packages/ember/version.ts

```diff
@@ -0,0 +1,2 @@
+// this file gets replaced with the real value during the build
+export default 'VERSION_GOES_HERE';
```

Comment:
By having a real file on disk, we get to eliminate hacks around the types and we avoid crashing if you try to directly evaluate in node, without a build.